app/services/rummager.rb

```diff
@@ -8,8 +7,7 @@ def self.rummager
 
   class Rummager
     def search(args)
-      results = rummager.search(args)
-      SearchMutator.mutate_search(results)
+      rummager.search(args)
```

Comment:
Yeh, I think it might be useful to have it in this form for now as we might find it useful.